.github/workflows/ci.yml

```diff
@@ -93,6 +95,21 @@ jobs:
       - name: Test Rust Host SDK
         run: LD_LIBRARY_PATH=/usr/local/lib cargo test --release -p ${{ env.RUST_SDK_CRATE }}
 
+      - name: Setup Elixir Host SDK
+        uses: erlef/setup-beam@v1
+        with:
+          experimental-otp: true
+          otp-version: '25.0.4'
+          elixir-version: '1.14.0'
+
+      - name: Test Elixir Host SDK
+        run: |
+         cd elixir &&
+         mix deps.get &&
+         mix deps.compile &&
+         mix compile &&
```

Comment:
Github actions is set up to fail on a per-line basis here, so I think you can leave out the `&&`. Curious though if you tried that and it didn't work!

Reply:
Yeah, i did this back when i was debugging something. Will change it back.